skipper/runner.py

```diff
@@ -56,6 +57,7 @@ def _run_nested(fqdn_image, environment, command, interactive, name, net, volume
     user_id = os.getuid()
     docker_cmd += ['-e', 'SKIPPER_USERNAME=%(user)s' % dict(user=user)]
     docker_cmd += ['-e', 'SKIPPER_UID=%(user_id)s' % dict(user_id=user_id)]
+    docker_cmd += ['-e', 'HOME=%(homedir)s' % dict(homedir=homedir)]
```

Comment:
I'm not sure what all the implications of this change, but I guess it's ok since it doesn't affect the product